en/docs/design/rate-limiting/advanced-topics/configuring-rate-limiting-api-gateway-cluster.md

```diff
@@ -34,6 +58,116 @@ Follow the instructions below to configure the Redis server with WSO2 API Manage
     'throttling.distributed.counter.type' = "redis"
     ```
 
-### Step 3 - Start the WSO2 API-M server
+#### Step 3 - Start the WSO2 API-M server
 
 For more information, see [Running the API Manager Runtime]({{base_path}}/install-and-setup/install/installing-the-product/running-the-api-m/).
+
+## 2. Async-Sync Hybrid Mode
+
+In this mode, throttling related parameters (counters and timestamps) are synced in-between gateway nodes and the Redis
+server both asynchronous and synchronous mode. Within a time window, up until an evenly allocated local quota of
+requests, throttle parameters are evaluated and processed locally only, which demonstrates an asynchronous behaviour.
+
+When running these throttle-param processing tasks, local quota value is calculated as below.
+> Local Quota (LQ) = (T - X%) / GW  
+<small>_T :  Throttle Limit (i.e. 100 tps)     
+X : Percentage of the buffer quota    
+GW : Count of gateway nodes active_</small>
+
+Active gateway count of the gateway cluster is counted by each of the gateway nodes using the redis server once per
+each 10 seconds of duration.
+
+So a local quota of API requests is evenly distributed among the active gateway nodes. After the local quota is
+exhausted in any gateway node, the throttle-param syncing of succeeding requests on any of the gateway nodes in the
+cluster will be processed in a synchronous behaviour. So when the local quota value determined is less, the API requests will
+start to process in a synchronous behaviour sooner.
+
+This mode is recommended over the [Asynchronous Mode](#1-asynchronous-mode) if you need to have a better throttle processing precession.
+But it is important to note that, it is not possible to achieve 100% precision with rate limiting implementations for
+small time windows (i.e. 1 second) when the traffic is high. There is always a trade-off between accuracy and latency,
+which are taken into consideration. There might be a positive or negative spill-over percentage of requests. And also
+this spill-over percentage will be dependent on the various factors i.e. infrastructure, network latency, performance of
+redis server, etc.
+
+As noted about regarding the latency, once enabled this feature, there will be an impact on the latency of API requests.
```

Comment:
As mentioned regarding the latency, once you enable this feature, there will be an impact on the latency of API requests.